python/setup.py

```diff
@@ -0,0 +1,66 @@
+import os
+import subprocess
+import re
+from setuptools.command.sdist import sdist
+from setuptools import setup, Extension, find_packages
+from setuptools.command.build_ext import build_ext
+from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+
+def get_long_description():
+    with open("../README.md", "r", encoding="utf-8") as f:
+        return f.read()
+
+
+def get_fml_executable():
+    os = subprocess.check_output(["go", "env", "GOOS"]).strip().decode("utf-8")
+    return "../fml.exe" if os == "windows" else "../fml"
+
+def get_version():
+    version = os.environ.get("VERSION")
+    matches = re.findall(r'\d+\.\d+\.\d+', version)
+    return matches[0]
+
+def get_platform():
+    os = subprocess.check_output(["go", "env", "GOOS"]).strip().decode("utf-8")
+    arch = subprocess.check_output(["go", "env", "GOARCH"]).strip().decode("utf-8")
+    plat = f"{os}_{arch}"
+    if plat == "darwin_amd64":
+        return "macosx_10_13_x86_64"
+    elif plat == "darwin_arm64":
+        return "macosx_11_0_arm64"
+    elif plat == "linux_amd64":
+        return "manylinux1_x86_64"
+    elif plat == "linux_arm64":
+        return "manylinux1_aarch64"
```

Comment:
just checked this locally and this is required to install properly everywhere:
```suggestion
    elif plat == "linux_amd64":
        return "manylinux_2_17_x86_64.manylinux2014_x86_64.musllinux_1_1_x86.64"
    elif plat == "linux_arm64":
        return "manylinux_2_17_aarch64.manylinux2014_aarch64.musllinux_1_1_aarch64"
```

Reply:
I think this is not a valid name for a wheel

Reply:
correct, I edited the suggestion above